bmc-reverse-proxy/TAG

```diff
@@ -1 +1 @@
-0.1.2
```

Comment:
Just a comment; this is to update k8s client modules.
If we had updated the modules, bmc-reverse-proxy need not be updated because its image is scratch-based.